src/pages/docs/sdk/integrations/vue.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Vue'
+--- 
+
+It is available as an [npm package](https://www.npmjs.com/package/@microlink/vue)
+
+<Terminal>npm install @microlink/vue --save</Terminal>
+
+The Vue bundle is based on the Vanilla version, but exported as a CommonJS Vue plugin/component.
+
+Since it doesn't include the peer dependencies, you'll need to install those too:
+
+<Terminal>npm install react react-dom styled-components --save</Terminal>
+
+After that, you're ready to import the Microlink SDK into your Vue project, either globally or on-demand.
+
+### Globally
+
+##### App.js
+```js
+import Vue from 'vue'
+import Microlink from '@microlink/vue'
+
+Vue.use(Microlink)
+```
+
+##### MyComponent.vue
+```vue
+<template>
+  <Microlink url="https://microlink.io" />
+</template>
+```
+
+### On-demand
+
+```vue
+<template>
+  <Microlink url="https://microlink.io" />
+</template>
+
+<script>
+import { Microlink } from '@microlink/vue'
+
+export default {
+  name: 'MyComponent',
+  components: { Microlink }
+}
+</script>
+```
+
```

Comment:
@Kikobeats maybe we can add a codesandbox link/iframe with a demo of a Vue environment? Same for Vanilla, and we could add an Angular integration page too.

Reply:
totally!

In the past, I added it as part of left panel, but now that we have a CodeSandbox per every integration is a worth it thing to add part of every integration doc.